src/domain/app/apollo/apolloClient.ts

```diff
@@ -1,22 +1,42 @@
+import {
+  ApolloClient,
+  ApolloLink,
+  HttpLink,
+  InMemoryCache,
+} from '@apollo/client';
+import { onError } from '@apollo/client/link/error';
 import * as Sentry from '@sentry/browser';
-import { InMemoryCache } from 'apollo-cache-inmemory';
-import { ApolloClient } from 'apollo-client';
-import { ApolloLink } from 'apollo-link';
-import { onError } from 'apollo-link-error';
-import { HttpLink } from 'apollo-link-http';
 import get from 'lodash/get';
 
 const cache = new InMemoryCache({
-  cacheRedirects: {
+  typePolicies: {
     Query: {
-      keywordDetails: (_, args, { getCacheKey }) => {
-        return getCacheKey({ __typename: 'Keyword', id: args.id });
-      },
-      placeDetails: (_, args, { getCacheKey }) => {
-        return getCacheKey({ __typename: 'Place', id: args.id });
+      fields: {
+        event(_, { args, toReference }) {
+          return toReference({
```

Comment:
if the cache is replaced, isn't the mergeIntoStore: true needed?

Reply:
Actually we are already fetching from cache here, not sure if adding that single keyword to cache is needed.